test/xfail.json

```diff
@@ -0,0 +1,500 @@
+{
+    "experimental": {
+        "uncategorised": {
+            "33": true,
+            "34": true,
+            "35": "skip",
```

Comment:
Looks like it's a decorator on the class.

`class Foo { @foo bar() {} }`

the ClassBody parser goes into an infinite loop